manifests/agent/yum_repos.pp

```diff
@@ -0,0 +1,11 @@
+class classroom::agent::yum_repos {
+  $classroom::yum_repos.each |$repo, $value| {
```

Comment:
oops this is a typo.  I fixed it on my testing machine, just a sec. I'll push it up.